src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -268,17 +295,62 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                             _ => throw new ArgumentException($"Unsupported style {ConfigurationSource.PhaseDisplayStyle}")
                         };
 
-                        return FormattableString.Invariant($@"
-                            <tr>
-                                <td>$\left|{basisLabel}\right\rangle$</td>
-                                <td>${amplitude.Real:F4} {(amplitude.Imaginary >= 0 ? "+" : "")} {amplitude.Imaginary:F4} i$</td>
+                        count = count + 1;
+
+                        //different options for displaying measurement style
+    
+                        var measurementCell = ConfigurationSource.MeasurementDisplayStyle switch
+                        {
+                            MeasurementDisplayStyle.None =>FormattableString.Invariant($@"
+                                <td> 
+                                    
+                                </td>
+                            "),
+                            MeasurementDisplayStyle.BarOnly => FormattableString.Invariant($@"
                                 <td>
                                     <progress
                                         max=""100""
                                         value=""{System.Math.Pow(amplitude.Magnitude, 2.0) * 100}""
                                         style=""width: 100%;""
                                     >
                                 </td>
+                            "),
+                            MeasurementDisplayStyle.BarAndNumber => FormattableString.Invariant($@"
+                                <td>
+                                    <progress
+                                        max=""100""
+                                        value=""{System.Math.Pow(amplitude.Magnitude, 2.0) * 100}""
+                                        style=""width: 70%;""
+                                    > 
+                                    <td>
+                                    <p id=""round${count}""> 
+                                    <script>
+                                    var num = {System.Math.Pow(amplitude.Magnitude, 2.0) * 100};
+                                    num = num.toFixed(2);
```

Comment:
This will be really nice! Just a suggestion: for the numerical display, perhaps we shouldn't multiply by 100, so that the users just see the probabilities in decimal form? And if we do that, I suspect it will be useful to show more than 2 digits after the decimal point. I'm not sure what is a good number, maybe 4 or 6?

Reply:
That could even be configurable, actually, with something like 4 as a default.

Reply:
Created a new config option to set the measurement probability precision, default is 4. Also got rid of multiplication by 100 when viewing the measurement probability as a decimal.

Reply:
@Reemeela2121 it looks like maybe the change you're referring to didn't get applied properly? I don't see it here.